unity-renderer/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/Shaders/FlibBook.shader

```diff
@@ -0,0 +1,94 @@
+// https://unitycoder.com/blog/2018/11/30/sprite-sheet-flip-book-shader/
+
+Shader "UnityLibrary/Sprites/FlipBook (Cutout)"
```

Comment:
The file is named `FlibBook` we should rename to `FlipBook`

Reply:
Ups, good catch! Changed in https://github.com/decentraland/unity-renderer/pull/909/commits/e156b2009b323aad335c2c628712dd6757913e2d. Thanks!